README.md

```diff
@@ -39,9 +44,19 @@ flags menu:
 - transform/warp                // Toggle flag for transforming/warping
 - wolf sense                    // Toggle flag for having wolf sense
 
+dungeon flags menu:
+- have map                      // Toggle dungeon map for current dungeon
+- have compass                  // Toggle compass for current dungeon
+- have boss key                 // Toggle boss key for current dungeon
+- have small keys               // Toggle small keys for current dungeon
+- miniboss dead                 // Toggle miniboss defeated state
+- boss dead                     // Toggle boss defeated state
+- clear flags*                  // Clear all dungeon flags
+
 inventory menu:
-- item wheel                    // Can set the 24 item wheel slots to any item
-- pause menu (TBD)              // Can modify the pause menu collection (not implemented yet)
+- item wheel                    // Set the 24 item wheel slots to any item
+- pause menu                    // Modify the pause menu collection
+- amounts menu                  // Modify ammo / collectible amounts
```

Comment:
This is really minor, but there probably should not be spaces around the "/" to be consistent.